data/mods/Magiclysm/enchanted/enchanted_tokens.json

```
@@ -0,0 +1,179 @@
+[
+  {
+    "abstract": "mtoken",
+    "name": "magic token",
+    "type": "GENERIC",
+    "category": "other",
+    "weight": "8 g",
+    "volume": 0,
```

```suggestion
    "volume": "1 ml",
```
an american dollar coin is about 1.102 mL, [see this chart](https://www.usmint.gov/learn/coin-and-medal-programs/coin-specifications)

---

Ah, good. I copied this from the existing coin item in the game.